src/Symfony/Component/Translation/Loader/CsvFileLoader.php

```diff
@@ -57,6 +60,11 @@ public function setCsvControl(string $delimiter = ';', string $enclosure = '"',
     {
         $this->delimiter = $delimiter;
         $this->enclosure = $enclosure;
-        $this->escape = $escape;
+        $escape = func_num_args() > 2 ? func_get_arg(2) : null;
+        if (null !== $escape) {
```

Comment:
this looks unfinished: let's remove the argument? why use func_* function then?
also, this is going to be triggered by L41, which means a self-deprecation

Reply:
I admit that I forgot what happened here, likely an artifact from a previous try. This should be fixed now.

Also, the parameter cannot be removed cannot be removed as it's a public method.